301-metadata-repositories.md

```diff
@@ -8,6 +8,8 @@ weight: 301
 * [Metadata repositories](#metadata-repositories)
 * [Minimum viable product (MVP)](#minimum-viable-product-mvp)
   * [Security analysis of MVP](#security-analysis-of-mvp)
+* [Maximum security product (MSP)](#maximum-security-product-msp)
```

Comment:
I will have to agree with @squillace and suggest we don't rely on even more acronyms.

Reply:
What do you both suggest? It's hard to refer to something over and over again w/o acronyms...

Reply:
resolved with https://github.com/cnabio/cnab-spec/pull/356/commits/16e9ea5acc7896d2d45b5b5bd61aca44232ff50a